components/TechnologyDropdown/styles.ts

```diff
@@ -72,6 +72,12 @@ export const ButtonWithIconStyles = styled.div`
   cursor: pointer;
 `;
 
+export const FilterIconStyles = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+`;
```

Comment:
Hmm previously only `ButtonWithIconStyles` had the `onClick` property, but I had to manually add it for each of the sub-components in this sprint because of the temporary `clearFilters` method on the exit button. I don't think this change will be permanent, but I do agree that we should look into making `ButtonWithIconStyles` extend a button instead of a div.